src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -312,6 +324,8 @@ fluid.textToSpeech.queueSpeech = function (that, text, interrupt, options) {
         that.cancel();
     }
 
+    options = options || {};
+    options.voice = that.getVoiceByLang(options.lang);
```

Comment:
I think this should be something like the following, so that a supplied voice in the options would be used, and the getVoice would be a fallback. 

```suggestion
    options.voice ??= that.getVoiceByLang(options.lang);
```